python/arcticdb/toolbox/library_tool.py

```diff
@@ -136,3 +137,23 @@ def read_to_keys(
         """
         df = self.read_to_dataframe(key)
         return self.dataframe_to_keys(df, id if id is not None else key.id, filter_key_type)
+
+
+    def read_descriptor(self, key: VariantKey) -> StreamDescriptor:
+        """
+        Gives the <StreamDescriptor> for a Variant key. The Stream Descriptor contains the <FieldRef>s for all fields in
+        the value written for that key.
+
+        E.g. an Index key will have fields like 'start_index', 'end_index', 'creation_ts', etc.
+        """
+        return super().read_descriptor(key)
```

Comment:
You can add docstrings directly to the method definition in the Python bindings:
https://github.com/man-group/ArcticDB/blob/master/cpp/arcticdb/version/python_bindings.cpp#L151-L175